Packages/vcs/vcs/vector.py

```diff
@@ -507,14 +522,17 @@ def _setlinecolor(self, value):
         self._linecolor = value
     linecolor = property(_getlinecolor, _setlinecolor)
 
-    def _getline(self):
-        return self._line
+    def _getlinetype(self):
+        return self._linetype
 
-    def _setline(self, value):
+    def _setlinetype(self, value):
         if value is not None:
-            value = VCS_validation_functions.checkLineType(self, 'line', value)
-        self._line = value
-    line = property(_getline, _setline)
+            value = VCS_validation_functions.checkLineType(self, 'linetype', value)
+        self._linetype = value
+    linetype = property(_getlinetype, _setlinetype)
+
+    def setLineAttributes(self, line):
```

Comment:
@danlipsa @aashish24 speaking of doc strings I asked our interns here to atually add them to the propertied objects as well. I will be a nice addition